src/containers/Validators/VotingTab.tsx

```diff
@@ -0,0 +1,113 @@
+import { useTranslation } from 'react-i18next'
+import { FC } from 'react'
+import axios from 'axios'
+import { useQuery } from 'react-query'
+
+import { ValidatorSupplemented } from '../shared/vhsTypes'
+import { SimpleRow } from '../shared/components/Transaction/SimpleRow'
+import {
+  FETCH_INTERVAL_ERROR_MILLIS,
+  FETCH_INTERVAL_VHS_MILLIS,
+  SERVER_ERROR,
+  renderXRP,
+} from '../shared/utils'
+import { useAnalytics } from '../shared/analytics'
+import { XRP_BASE } from '../shared/transactionUtils'
+
+import './votingTab.scss'
+
+export const VotingTab: FC<{
+  validatorData: ValidatorSupplemented
+  network: string | undefined
+}> = ({ validatorData, network }) => {
+  const { t } = useTranslation()
+  const { trackException } = useAnalytics()
+
+  const votedAmendments = new Set(
+    validatorData.amendments
+      ? validatorData.amendments.map((amendment) => amendment.id)
+      : [],
+  )
+  const { data } = useQuery<Array<{ id: string; name: string }>>(
+    ['fetchNetworkVotingData', network],
+    async () => fetchNetworkVote(network),
+    {
+      refetchInterval: (returnedData, _) =>
+        returnedData == null
+          ? FETCH_INTERVAL_ERROR_MILLIS
+          : FETCH_INTERVAL_VHS_MILLIS,
+      refetchOnMount: true,
+      enabled: !!network,
+    },
+  )
+
+  function fetchNetworkVote(networkID: string | undefined) {
+    const url = `${process.env.VITE_DATA_URL}/amendments/vote/${networkID}`
+    return axios
+      .get(url)
+      .then((resp) => resp.data)
```

Comment:
combine these two thens.  Also might be good to guard this incase this nesting is not there